src/server-code/main-server.js

```diff
@@ -161,39 +160,12 @@ export default function runMainServer(serverFalcorModel) {
   });
 
   if (isStaging) {
-    app.get('/login', (req, res) => {
-      match({ routes, location: req.url },
-        (error, redirectLocation, renderProps) => {
-          if (error) {
-            res.status(500).send(error.message);
-          } else if (redirectLocation) {
-            res.redirect(302, redirectLocation.pathname + redirectLocation.search);
-          } else if (renderProps) {
-            renderApp(renderProps, true).then((html) => {
-              res.status(200).send(html);
-            }).catch((err) => {
-              if (process.env.NODE_ENV !== 'production') {
-                console.error('Failed to render: ', req.url); // eslint-disable-line no-console
-                console.error(err.stack || err); // eslint-disable-line no-console
-                res.status(500).send(err.stack || err);
-              } else {
-                res.status(500).send('There was an error while serving you this webpage.' +
-                  ' Please contact The Gazelle team and tell them this link is broken. We hope' +
-                  ' to fix it soon. Thank you.');
-              }
-            });
-          } else {
-            res.status(404).send('Not Found');
-          }
-        }
-      );
+    app.get('/robots.txt', (req, res) => {
+      res.status(200).type('txt').send(nothingAllowedRobotsTxt);
     });
-    app.get(/(?!\/login)/, (req, res) => {
-      res.redirect(307, `/login?url=${req.url}`);
-    });
-  } else {
-    app.get('*', (req, res) => {
-      match({ routes, location: req.url },
+  }
+  app.get('*', (req, res) => {
+    match({ routes, location: req.url },
         (error, redirectLocation, renderProps) => {
```

Comment:
This part is now wrongly indented, linting should've caught that really. (you should move in this block from line 197 to 220 by one tab)